federation/pkg/dnsprovider/providers/aws/route53/route53_test.go

```diff
@@ -0,0 +1,235 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package route53
+
+import (
+	"flag"
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
```

Comment:
nit: At other places, we put a line break between these different "types" of dependencies.
Also we put k8s.io ones before github.com.
Example: https://github.com/kubernetes/kubernetes/blob/master/pkg/master/master.go